doc/cephadm/operations.rst

```diff
@@ -402,18 +402,25 @@ identifying anomalies.
 
 Client keyrings and configs
 ===========================
-
+Starting from versions 16.2.10 (Pacific) and 17.2.1 (Quincy) in addition to
+the default location ``/etc/ceph/`` cephadm also stores config and keyring
+files in the ``/var/lib/ceph/<fsid>/config`` directory.
```

Comment:
I feel like this paragraph fits better after the `Cephadm can distribute copies of . . .` paragraph than before it